src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+/**
+ * Represents a user interface to interact with the user.
+ */
+public class Ui {
+
+    public Ui () {}
+
+    /**
+     * Prints the greeting message when starting up the project.
+     */
+    protected String greeting() {
+        String str = "-----------------------------------\n"
+                + "Hello! I'm Cat\n"
+                + "What can I do for you?\n"
+                + "-----------------------------------\n";
+        return str;
+//        System.out.println("-----------------------------------");
+//        System.out.println("Hello! I'm Cat");
+//        System.out.println("What can I do for you?");
+//        System.out.println("-----------------------------------");
+//        System.out.println();
+    }
+
+    /**
+     * Prints the error message when the task that user is looking for does not exist in the list.
+     */
+    protected String taskNotExist() {
+        return "This task does not exist in your list\n";
+        //System.out.println("This task does not exist in your list");
```

Comment:
Consider removing unwanted code (including rest of the print statements in Ui class)